### ISSUE TITLE
Batch convert links

### ISSUE THREAD

**Issue by visualcurrent** [agathauy/wikilinks-to-mdlinks-obsidian#4]
This plugin is so great! I'm posting a feature request, not an issue.

Would be relatively easy to add an option to batch convert all links. It would be amazing to convert all links in a page, folder, or vault at one time.

**Comment by agathauy** [comment]
On this soon!

**Comment by LamFTS** [comment]
looking forward to, need this urgently

**Comment by zhangzz66** [comment]
Looking forward to batch conversion of vault

**Comment by Archie-2021** [comment]
Plus one for this one. It could get the desired input by selection

**Comment by BuaaLzd** [comment]
looking forward to batch convertion.

**Comment by ZJsheep** [comment]
Looking forward to that. One year has passed and is there still any hope?

**Comment by Archie-2021** [comment]
> Looking forward to that. One year has passed and is there still any hope?

I don't think that be an very regex. if this plugin is inactive someone else can make a new one. it is such a small project with great rewards

**Comment by DrLeucine** [comment]
@agathauy  Are you still working on this by any chance?

**Comment by Archie-2021** [comment]
There is this plugin that do this: obsidian://show-plugin?id=obsidian-link-converter
